assets/scss/app.scss

```diff
@@ -1,5 +1,5 @@
 $assetPath: '/public';
-
+$red: #d4351c;
```

Comment:
I have no idea where we get this from in the main app, but I can't get css to build here without it.

Reply:
I think it's defined in Components for the diff:
https://github.com/UKHomeOffice/asl-components/blob/fc8aca46fdc3fc2c0fc3bf9562ea669776ffe28f/src/diff-text/index.scss#L1